akka-docs/src/main/paradox/testing-typed.md

```diff
@@ -29,4 +29,133 @@ Maven
     ```
     @@@
     
-TODO
\ No newline at end of file
+Testing can either be done asynchronously using a real `ActorSystem` or synchronously using the `BehaviousTestKit`.  
```

Comment:
perhaps clarify what we mean with synchronously in this context.

it's running in a single thread, the thread executing the test.

Also some guidelines of when to use what. BehaviousTestKit is for unit testing of a single behavior. Interactions between several actors are better tested with async testkit.

Also limitations of sync testing, e.g. spawning futures.

Reply:
:+1: